corehq/apps/app_manager/views/forms.py

```diff
@@ -576,6 +576,7 @@ def get_form_view_context_and_template(request, domain, form, langs, messages=me
             {'instanceId': instance.instance_id, 'instancePath': instance.instance_path}
             for instance in form.custom_instances
         ],
+        'can_preview_form': request.couch_user.has_permission(domain, 'edit_data')
```

Comment:
@benrudolph just to keep the intent clear. `allow_cloudcare` seems picking from the form instance type whereas `can_preview_form` would be user permission.  Don't mind combining them if that looks better.